docs/source/tracking.rst

```diff
@@ -180,6 +178,78 @@ and the `LocalArtifactRepository <https://github.com/mlflow/mlflow/blob/master/m
+
+MLflow's Tracking Server supports utilizing the host as a proxy server for operations involving artifacts.
+Once configured with the appropriate access requirements, an administrator can start the tracking server to enable
+assumed-role operations involving the saving, loading, or listing of model artifacts, images, documents, and files.
+This eliminates the need to allow end users to have direct path access to a remote object store (e.g., s3, adls, gcs, hdfs) for artifact handling and eliminates the
+need for an end-user to provide access credentials to interact with an underlying object store.
+
+.. figure:: _static/images/scenario_5.png
+
+Enabling the Tracking Server to perform proxied artifact access in order to route client artifact requests to an object store location:
+
+ * **Part 1a and b**:
+
+  * The MLflow client creates an instance of a `RestStore` and sends REST API requests to log MLflow entities
+  * The Tracking Server creates an instance of an `SQLAlchemyStore` and connects to the remote host for inserting
+    tracking information in the database (i.e., metrics, parameters, tags, etc.)
+
+ * **Part 1c and d**:
+
+  * Retrieval requests by the client return information from the configured `SQLAlchemyStore` table
+
+ * **Part 2a and b**:
+
+  * Logging events for artifacts are made by the client using the ``HttpArtifactRepository`` to write files to MLflow Tracking Server
+  * The Tracking Server then writes these files to the configured object store location with assumed role authentication
+
+ * **Part 2c and d**:
+
+  * Retrieving artifacts from the configured backend store for a user request is done with the same authorized authentication that was configured at server start
+  * Artifacts are passed to the end user through the Tracking Server through the interface of the ``HttpArtifactRepository``
+
+.. warning::
+    The MLflow artifact proxied access service enables users to have an *assumed role of access to all artifacts* that are accessible to the Tracking Server.
+    Administrators who are enabling this feature should ensure that the access level granted to the Tracking Server for artifact operations meets with
+    any security requirements prior to enabling the Tracking Server to operate in a proxied file handling role.
+
+.. _scenario_6:
+
+Scenario 6: MLflow Tracking Server used exclusively as proxied access host for artifact storage access
+------------------------------------------------------------------------------------------------------
+
+MLflow's Tracking Server can be used in an exclusive artifact proxied access file handling role. The configuration at server start by adding the flag
+``--artifacts-only`` will restrict a Tracking Server instance to be used as a bastion host to an underlying file store without allowing for Tracking Server
+functionality apart from artifact handling. See: :ref:`artifact_only_mode` for more details.
```

Comment:
"bastion host" is somewhat bespoke terminology. Can we simplify the phrasing here? 

```suggestion
MLflow's Tracking Server can be used in an exclusive proxied artifact handling role. Specifying the
``--artifacts-only`` flag restricts an MLflow server instance to only serve artifact-related API requests by
proxying to an underlying object store.
```